docs/bib/use_citations.bib

```diff
@@ -75,12 +75,16 @@ @article{Alguero:2020grj
 }
 
 % 2020-08-14
-@Booklet{ATLAS-CONF-2020-046,
-    author         = "{ATLAS Collaboration}",
-    title          = "{Search for new phenomena in events with two opposite-charge leptons, jets and missing transverse momentum in \(pp\) collisions at \(\sqrt{s} = 13\,\text{TeV}\) with the ATLAS detector}",
-    howpublished   = "{ATLAS-CONF-2020-046}",
-    url            = "https://cds.cern.ch/record/2728056",
-    year           = "2020",
+@article{Aad:2021hjy,
+    author = "Aad, Georges and others",
+    collaboration = "ATLAS",
+    title = "{Search for new phenomena in events with two opposite-charge leptons, jets and missing transverse momentum in $pp$ collisions at $\sqrt{s} = 13$ TeV with the ATLAS detector}",
+    eprint = "2102.01444",
+    archivePrefix = "arXiv",
+    primaryClass = "hep-ex",
+    reportNumber = "CERN-EP-2020-243",
+    month = "2",
+    year = "2021"
```

Comment:
```suggestion
    year = "2021",
    journal = ""
```

The error in the CI gives
```
Extension error (sphinxcontrib.bibtex.domain):
Handler <function env_updated at 0x7fab8ff089d0> for event 'env-updated' threw an exception (exception: missing journal in Aad:2021hjy)
```

because it is expecting a `journal` field for `@article`.

Reply:
I copied it from inspirehep -- so maybe that's a problem with them?